src/components/comment/comment.ts

```diff
@@ -0,0 +1,49 @@
+export const initCommentReplyForm = (comment: HTMLElement) => {
+  const replyButton = comment.querySelector<HTMLButtonElement>(
+    '.js-comment-reply-button'
+  );
+  const replyForm = comment.querySelector<HTMLFormElement>(
+    '.js-comment-reply-form'
+  );
+  // The cancel button is inside the Comment Form, so we need to reach inside
+  // that component to grab the button. This feels a little awkward since
+  // we're directly manipulating a child component, but this is a lot simpler
+  // than other potential solutions. Other options included:
+  // - Having the Comment Form emit a `cancel` event
+  // - Having the Comment Form have a block that you could pass a button into
+  const cancelButton =
+    comment.querySelector<HTMLButtonElement>('.js-cancel-reply');
+
+  const show = () => {
+    comment.classList.add('is-replying');
+    replyButton?.setAttribute('hidden', '');
```

Comment:
@Paul-Hebert Instead of using optional chaining for accessing all these elements, could we check that they all exist near the top? Or does there need to be partial functionality if only some exist?

```js
const replyButton = ...
const replyForm = ...
const cancelButton = ...

if (!replyButton || !replyForm || !cancelButton) return { destroy: () => {} }
```

Reply:
Sure :+1: